script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -291,27 +310,30 @@ Why isn't she pulling the trigger?
 [sha:19a537d92d13e141eca341bb3ca88b3542d185b7]{
 -- Page 403, Offset 34441.
 -- 俺を見つめてくる目は虚ろだ。
-Her eyes that gaze upon me seem empty.
+Her eyes seem oddly empty as she gazes down at me.
 }
 [sha:28602ce521fda2d07dbf50afc62eb614033a9b2b]{
 -- Page 404, Offset 34442.
 -- 先輩が時折見せる、感情のない瞳。それは、この人が冷酷な人という訳じゃなくて。
-Every once in a while, she has this emotionless look in her eyes, but that doesn't mean that Senpai is a cold-hearted person.
+This isn't the first time I've seen that emotionless look in her eyes.
+ That's not to say she's cold-hearted, or anything like that.
 }
 [sha:fca1470e8b6e8f19fa109e88f161cedd81131ae3]{
 -- Page 404, Offset 34443.
 -- 最後の最後の<局面|ライン>で自分自身を騙せないから、結局、自分の感情を殺しているだけ。
-It's just that she's incapable of deceiving herself until the very end, so all she can do is suppress her own emotions instead.
+It's just that, try as she might, she's incapable of truly believing the lies she tells herself.
+ Casting off her emotions is her only means of escape.
```

Comment:
I intentionally rendered it this way to match the prologue - 
> 「志貴、心にもない事は言わないこと。自分自身も<騙|だま>せないような嘘は、聞いている方を不快にさせるわ」
> ...Telling lies that even you yourself can't believe will only make whoever hears you sad

Reply:
This fucking prologue is tied to every scene in the game, I swear to god. I'm so tired. 
I still think the connection here is kinda important... what to change, what to change...

>It's just that, try as she might, she's incapable of truly believing the lies she tells herself.
 Casting off her emotions is her only means of escape.

>I've only just realized that whenever she gets that look in her eyes, it isn't because she's lying to me――it's because she wants to believe in the lie herself.

Or...

>Telling lies that not even you'd be deceived by just feels bad for the one listening.